naturalcrit/homebrewery
# V4 proposed aligned paragraph tokens

## Description

This implements the suggested syntax for V4 paragraph text alignment tokens from https://github.com/naturalcrit/homebrewery/discussions/2879

## Related Issues or Discussions

https://github.com/naturalcrit/homebrewery/discussions/2879

## QA Instructions, Screenshots, Recordings

- insert `:- ` in front of a paragraph. It should align left. You should see a class of mdParagraphJustifyLeft on the paragraph tag
- insert `-: ` in front of a paragraph. It should align right. You should see a class of mdParagraphJustifyRight on the paragraph tag
- insert `:-: ` in front of a paragraph. It should align center. You should see a class of mdParagraphJustifyCenter on the paragraph tag

### Reviewer Checklist
